Gopkg.toml

```diff
@@ -38,6 +38,10 @@ required = [
   name = "k8s.io/apiextensions-apiserver"
   branch = "release-1.14"
 
+[[constraint]]
```

Comment:
Can you add a note here like what you did in the other PR?
TODO: remove when official 0.4.0 tagged

Reply:
0.4.0 is released so I'll just update to use that instead :)